### PR TITLE
fix: Use correct handler for SvelteKit getting started

### LINKED ISSUE #178: Fix SvelteKit get started code
The following step is incorrect in the SvelteKit get started guide:

https://docs.arcjet.com/get-started?f=sveltekit#3-add-rules

The file incorrectly exports `handle()` when it should export the req method function.

### PR DESCRIPTION
**Pull request by morekid**
Closes #178

### CODE REVIEW

File: src/snippets/get-started/sveltekit/Step3.ts [hunk 1]
```diff
@@ -45,5 +39,15 @@ export async function handle({
     }
   }
 
-  return resolve(event);
+  return new Response(
+    JSON.stringify({
```
Comment (morekid) [inline src/snippets/get-started/sveltekit/Step3.ts:43]: what do we want to return as default?

File: src/snippets/get-started/sveltekit/Step3.js [hunk 2]
```diff
@@ -41,3 +41,30 @@ export async function handle({ event, resolve }) {
...
+      return error(403, "No Bots Allowed");
+    } else {
+      return error(403, "Forbidden");
+    }
+  }
+
+  return new Response(
+    JSON.stringify({
+      event: event,
+    }),
```
Comment (blaine-arcjet) [inline src/snippets/get-started/sveltekit/Step3.js:62]: `{ message: "Hello World" }`

However, this should use the json helper in sveltekit:
```
import { json } from '@sveltejs/kit';
```
It looks like this is the same import as `error`

File: src/snippets/get-started/sveltekit/Step3.ts (hunk 1 above)
Comment (blaine-arcjet) [reply src/snippets/get-started/sveltekit/Step3.ts:43]: See above. You shouldn't need the additional argument for 200/content-type if you use the helper.

**Review by blaine-arcjet — CHANGES_REQUESTED** [review]
The `export async function handle({ event, resolve }) {` function needs to be completely removed.

**Review by blaine-arcjet — APPROVED** [review]

File: src/snippets/get-started/sveltekit/Step3.ts (hunk 1 above)
Comment (morekid) [reply src/snippets/get-started/sveltekit/Step3.ts]: should be taken care of now

File: src/snippets/get-started/sveltekit/Step3.js (hunk 2 above)
Comment (morekid) [reply src/snippets/get-started/sveltekit/Step3.js]: updated

**Pull request merged**